content/lua/concepts/coroutines/terms/wrap/wrap.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'wrap()'
+Description: 'Returns a function that can resume a coroutine, acting like a thread but returning a function.'
+Subjects:
+  - 'Computer Science'
+  - 'Lua Programming'
+Tags:
+  - 'Methods'
+  - 'Coroutines'
```

Comment:
I don't believe coroutines is currently part of our tags list. Please make the addition to the list or remove it here